spec/request/service_credential_bindings_spec.rb

```diff
@@ -1068,6 +1072,11 @@ def check_filtered_bindings(*bindings)
             get "/v3/service_credential_bindings/#{@binding_guid}", {}, admin_headers
             expect(last_response).to have_status_code(200)
             expect(parsed_response).to match_json_response(binding_response)
+
+            binding = VCAP::CloudController::ServiceBinding.last
+            expect(binding).to be
```

Comment:
What does this line do?

Reply:
It checks that it is not nil, similar to `expect(binding).to_not be_nil`
On failure it errors like this:
```
Failure/Error: expect(binding).to be
       expected nil to evaluate to true
```

I can see that it's not widely used, so I will change the occurrences to `to_not be_nil` just to not introduce more ways to do that same thing